src/Exporter/WebformFormatted/PaymentPropertySelector.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\campaignion_csv\Exporter\WebformFormatted;
+
+use Drupal\little_helpers\Webform\Submission;
+
+/**
+ * Select one of the submission properties.
+ */
+class PaymentPropertySelector implements SelectorInterface {
+
+  protected $property;
+  protected $cid;
+
+  /**
+   * Create a new instance from an info-array.
+   *
+   * @param array $info
+   *   The info array. The following propertys are in use:
+   *   - property: Name of the payment property. Also works with paths like
+   *     `method_data.account`.
+   *   - cid: A paymethod selector cid.
+   */
+  public static function fromInfo(array $info) {
+    $info += [
+      'property' => 'pid',
+      'cid' => NULL,
```

Comment:
There might be more then one paymethod selector on a form and we want the other one and happen to know it's `cid`?
Since `$submission->payments` are keyed by `cid` it suggested itself to offer a way to choose a payment by `cid`.

Reply:
I know that this is the structure of  `$submission->payments`. There is however no easy way to ensure consistent `cid`s across webforms. So this is probably not useful for exports. We could use a form_key again for this … or we do turn this around and export payments (with data from the payment context) instead of submissions.

Reply:
Happy to just remove the `cid` argument?

Reply:
Totally fine with that ;-)